data/interfaces.txt

```diff
@@ -99,7 +99,7 @@ color "missile dangerous" 1. .1 .1 1.
 color "radar player" .2 1. 0. 0.
 color "radar friendly" .4 .6 1. 0.
 color "radar unfriendly" .8 .8 .4 0.
-color "radar hostile" 1. .6 .4 0.
+color "radar hostile" 1. .3 .2 0.
```

Comment:
Messing around with this, I think this new color is a tad bit too bright. Turns out this color is used for more than just the radar; the targeted ship outline, targeted ship blips, and targeted planet blips also use this color. (And the hostile color is used for restricted planets that you have selected, which was a bit surprising.) While a bright color would work on just the radar, having this color used elsewhere was a bit jarring to me, so bringing the red component down a bit to lower the overall brightness. Those other uses of this color could potentially be split off, but that shouldn't be the responsibility of this PR.

I'm fine with the icon being slightly brighter and keeping its current color.
```suggestion
color "radar hostile" .85 .3 .2 0.
```